### PR TITLE
samples: ucaip sample batch 2 of 6

### PR DESCRIPTION
**Pull request by munkhuushmgl**
batch 2 of 6

### CODE REVIEW

**Comment by stephaniewang526** [comment]
Looks like environment variable `UCAIP_PROJECT_ID` from testing project. Please set it in Secret Manager and re-run ITs. Thanks.

File: samples/snippets/src/main/java/aiplatform/ImportDataImageClassificationSample.java [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+                  .build());
+
+      OperationFuture<ImportDataResponse, ImportDataOperationMetadata> importDataResponseFuture =
+          datasetServiceClient.importDataAsync(datasetName, importDataConfigList);
+      System.out.format(
+          "Operation name: %s\n", importDataResponseFuture.getInitialFuture().get().getName());
+      System.out.println("Waiting for operation to finish...");
+      ImportDataResponse importDataResponse = importDataResponseFuture.get(300, TimeUnit.SECONDS);
+
+      System.out.format("Import Data Image Classification Response: %s\n", importDataResponse);
```
Comment (stephaniewang526) [inline samples/snippets/src/main/java/aiplatform/ImportDataImageClassificationSample.java:84]: Can you verify that you dont need to call the `toString` method here? Printing out the object signature is probably not very useful.

File: samples/snippets/src/main/java/aiplatform/ImportDataImageObjectDetectionSample.java [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+                  .build());
+
+      OperationFuture<ImportDataResponse, ImportDataOperationMetadata> importDataResponseFuture =
+          datasetServiceClient.importDataAsync(datasetName, importDataConfigList);
+      System.out.format(
+          "Operation name: %s\n", importDataResponseFuture.getInitialFuture().get().getName());
+      System.out.println("Waiting for operation to finish...");
+      ImportDataResponse importDataResponse = importDataResponseFuture.get(300, TimeUnit.SECONDS);
+
+      System.out.format("Import Data Image Object Detection Response: %s\n", importDataResponse);
```
Comment (stephaniewang526) [inline samples/snippets/src/main/java/aiplatform/ImportDataImageObjectDetectionSample.java:83]: Same as above -- consider calling `toString()`

File: samples/snippets/src/main/java/aiplatform/ImportDataVideoObjectTrackingSample.java [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+                  .build());
+
+      OperationFuture<ImportDataResponse, ImportDataOperationMetadata> importDataResponseFuture =
+          datasetServiceClient.importDataAsync(datasetName, importDataConfigs);
+      System.out.format(
+          "Operation name: %s\n", importDataResponseFuture.getInitialFuture().get().getName());
+      System.out.println("Waiting for operation to finish...");
+      ImportDataResponse importDataResponse = importDataResponseFuture.get(300, TimeUnit.SECONDS);
+
+      System.out.format("Import Data Video Object Tracking Response: %s\n", importDataResponse);
```
Comment (stephaniewang526) [inline samples/snippets/src/main/java/aiplatform/ImportDataVideoObjectTrackingSample.java:81]: Same as above -- consider calling `toString()`.

**Review by stephaniewang526 — CHANGES_REQUESTED** [review]

File: samples/snippets/src/main/java/aiplatform/CreateTrainingPipelineImageObjectDetectionSample.java [hunk 4]
```diff
@@ -0,0 +1,235 @@
...
+      System.out.format(
+          "Feature Attributions Schema_uri: %s\n",
+          explanationMetadata.getFeatureAttributionsSchemaUri());
+
+      Status status = trainingPipelineResponse.getError();
+      System.out.println("Error");
+      System.out.format("Code: %s\n", status.getCode());
+      System.out.format("Message: %s\n", status.getMessage());
+
+      List<Any> detailsList = status.getDetailsList();
```
[the remark this answers is not in the document]
Comment (munkhuushmgl) [reply samples/snippets/src/main/java/aiplatform/CreateTrainingPipelineImageObjectDetectionSample.java:231]: I think it should be deleted.

File: samples/snippets/src/main/java/aiplatform/DeleteBatchPredictionJobSample.java [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+      String location = "us-central1";
+
+      BatchPredictionJobName batchPredictionJobName =
+          BatchPredictionJobName.of(project, location, batchPredictionJobId);
+
+      OperationFuture<Empty, DeleteOperationMetadata> operationFuture =
+          jobServiceClient.deleteBatchPredictionJobAsync(batchPredictionJobName);
+      System.out.format("Operation name: %s\n", operationFuture.getInitialFuture().get().getName());
+      System.out.println("Waiting for operation to finish...");
+      Empty deleteResponse = operationFuture.get(300, TimeUnit.SECONDS);
```
[the remark this answers is not in the document]
Comment (munkhuushmgl) [reply samples/snippets/src/main/java/aiplatform/DeleteBatchPredictionJobSample.java:62]: I just checked response, and its empty object. nothing being printed out of the response.
I will delete this response object.

**Review by stephaniewang526 — APPROVED** [review]

**Pull request merged**